docs/reference/modules/node.asciidoc

```diff
@@ -1,32 +1,304 @@
+duties and take away the ability to hold data, then you are left
+with a node that can only route requests, handle the search
+reduce phase, and distribute bulk indexing.
+
+Standalone client nodes provide large scale clusters a few
+benefits, by offloading some of the overhead associated with
+search and bulk indexing. Client nodes join the cluster and
+receive the full <<cluster-state,cluster state>>, like every
+other node, and they use the cluster state to route requests to
+the appropriate place(s). 
+
+WARNING: Unnecessarily adding too many client nodes to a cluster
+can increase the burden on the entire cluster because the
+elected master node must await acknowledgement of the cluster
+state from every node!
+
+Unlike master nodes and data nodes, the entire purpose of
+client nodes is to handle and route requests. This means that it
+is good practice to send them any type of request, although they
+are generally expected to be used for data-related requests.
+
+The benefit of the client nodes should not be overstated though.
+Client nodes benefit other nodes by offloading the distributed
+aspect of the request (e.g., the memory burden associated with
+collecting a large, distributed aggregation). The routing aspect
+is a convenience, but the focus on any node should be on the burden
+that they offload from other nodes.
+
+To create a standalone data node:
+
+[source,yaml]
+-------------------
+node.master: false <1>
+node.data: false <2>
+-------------------
+<1> Disable the `node.master` role (enabled by default).
+<2> Disable the `node.data` role (enabled by default).
+
+[float]
+[[node-configuration]]
+=== Node Configuration
+
+[float]
+[[node-configuration-disk]]
+==== Disk Configuration
+
+[cols="<,<,<",options="header",]
+|=======================================================================
+|Setting |Default |Description
+|`path.data` |`${ES_HOME}/data` |The place to store data (and metadata) for the
+node. This should be set uniquely per node.
+|`node.max_local_storage_nodes` |`50` |The number of nodes that can
+share the same `path.data` directory.
+|=======================================================================
+
+Every node should set its `path.data`, which, in the case of
+non-data nodes, allows the storage of metadata associated with
+the cluster. For data nodes, this is naturally where the data
+is stored.
+
+WARNING: Never run different node types from the same data
+directory. This can lead to unexpected data loss.
+
+To prevent `path.data` from being unexpectedly shared by
+any node, then you can set the
+`node.max_local_storage_nodes` setting.
+
+[source,yaml]
+-------------------
+node.max_local_storage_nodes: 1
+-------------------
+
+With this setting applied to each node, then the data
+directory cannot be accidentally shared and any node
+attempting to go above the set number (e.g., the second
+node) will be prevented from starting.
+
+NOTE: Running multiple nodes on the same machine is not
```

Comment:
NOTE: While useful for testing, running multiple nodes on the same machine in production...